targets/csharp-unityv2/make.js

```diff
@@ -0,0 +1,354 @@
+var path = require("path");
+
+exports.putInRoot = true;
+
+exports.makeCombinedAPI = function(apis, sourceDir, apiOutputDir){
```

Comment:
I don't like the fact that each of the 4 of these (makeCombinedAPI, makeClientAPI, makeServerAPI, makeAdminAPI) are so different from one another.  That indicates that we're doing too much custom stuff with each, and we need to make it more generic.  Wherever needed, we should just separate the target subfolders appropriately.
potentially 4 source folders, and a sharedSource folder, or even template folders, or whatever we need to do to make this sequence more straightforward and consistent.